sdk/nodejs/runtime/settings.ts

```diff
@@ -53,7 +55,25 @@ export interface Options {
 /**
  * options are the current deployment options being used for this entire session.
  */
-const options = loadOptions();
+let options = loadOptions();
+
+
+export function setMockOptions(mockMonitor: any, project: string, stack: string, preview: boolean | undefined) {
+    options = {
+        project: "project",
+        stack: "stack",
```

Comment:
possible to pass through `options.stack` here?  this hard-coded value is tripping up some of my tests on resources that interpolate the stack name in various places.  (which we set to "local" via `PULUMI_NODEJS_STACK` before running the tests).

Reply:
Ah, yeah--this is just a bug. Will fix.